aries-site/src/examples/components/layer/LayerStickyFormExample.js

```diff
@@ -0,0 +1,167 @@
+import React, { useContext, useState } from 'react';
+import {
+  Button,
+  Box,
+  CheckBox,
+  Form,
+  FormField,
+  Header,
+  Heading,
+  Layer,
+  ResponsiveContext,
+  Text,
+  TextInput,
+  TextArea,
+  Select,
+} from 'grommet';
+
+const options = ['Any', 'None', 'Any Scheduled'];
+
+export const LayerStickyFormExample = () => {
+  const [formValues, setFormValues] = React.useState({
+    monitorName: 'C2 Monitor',
+    desc: `Sustainable neutra echo park helvetica freegan
+    forage hashtag meditation kogi activated charcoal
+    pitchfork`,
+    monitorType: 'Rule',
+    frameworks: 'Any Scheduled',
+    severity: 'Any',
+    provider: 'Any',
+    account: 'Any',
+    ruleName: 'Any Scheduled',
+    email: 'jane@hpe.com',
+  });
+  const [open, setOpen] = useState(false);
+  const size = useContext(ResponsiveContext);
+  const onOpen = () => setOpen(true);
+  const onClose = () => setOpen(undefined);
+
+  const onFormChange = value => {
+    setFormValues(value);
+  };
+
+  // eslint-disable-next-line no-unused-vars
+  const onSubmit = ({ value, touched }) => {
+    // Your submission logic here
+  };
+  return (
+    <>
+      <Box align="start">
+        <Button label="Show me the Layer" onClick={onOpen} primary />
+      </Box>
+      {open && (
+        <Layer
+          position="right"
+          full={!['xsmall', 'small'].includes(size) ? 'vertical' : true}
+          modal
+          onClickOutside={onClose}
+          onEsc={onClose}
+        >
+          <Box
+            fill="vertical"
+            overflow="auto"
+            width={!['xsmall', 'small'].includes(size) ? 'medium' : undefined}
+          >
+            <Header
+              background="white"
+              sticky="scrollup"
```

Comment:
The layer is always staying sticky -- it's not scrolling away when I scroll down.